website/src/docs/hotchocolate/security/authorization.md

```diff
@@ -0,0 +1,356 @@
+---
+title: Authorization
+---
+
+import { ExampleTabs } from "../../../components/mdx/example-tabs"
+
+Authorization allows us to determine a user's permissions within our system. We can for example limit access to resources or only allow certain users to execute specific mutations.
+
+Authentication is a prerequisite of Authorization, as we first need to validate a user's "authenticity" before we can evaluate his authorization claims.
+
+[Learn how to setup authentication](/docs/hotchocolate/security/authentication)
+
+# Setup
+
+After we have successfully setup authentication, there are only a few things left to do.
+
+1. Register the necessary ASP.NET Core services
+
+```csharp
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddAuthorization();
+
+        // Omitted code for brevity
+
+        services
+            .AddGraphQLServer()
+            .AddAuthorization()
+            .AddQueryType<Query>();
+    }
+}
+```
+
+> ⚠️ Note: We need to call `AddAuthorization()` on the `IServiceCollection`, to register the services needed by ASP.NET Core, and on the `IRequestExecutorBuilder` to register the `@authorize` directive and middleware.
+
+2. Register the `UseAuthorization` middleware with the request pipeline
+
+```csharp
+public class Startup
+{
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        app.UseRouting();
+
+        app.UseAuthentication();
+        app.UseAuthorization();
+
+        app.UseEndpoints(endpoints =>
+        {
+            endpoints.MapGraphQL();
+        });
+    }
+}
+```
+
+# Usage
+
+At the core of authorization with Hot Chocolate is the `@authorize` directive. It can be applied to fields and types to denote that they require authorization.
+
+If the `@authorize` directive is specified on a type, it is applied to all fields of that type. Specified on an individual field the directive precedences the one on the type.
```

Comment:
```suggestion
If the `@authorize` directive is specified on a type, it is applied to all fields of that type. Specified on an individual field the directive overrules the one on the type.
```

I think it's worth mentioning here in an aside/note that it applies the type-level authorization by essentially annotating every field, so the authorization logic will be run once for each field.

cc: @michaelstaib

Reply:
Do you like this better?
![image](https://user-images.githubusercontent.com/45513122/124011154-fb2e9300-d9df-11eb-8ba6-c8805f5b6b68.png)